channels/us.m3u

```diff
@@ -826,26 +796,18 @@ https://dai2.xumo.com/amagi_hls_data_xumo-host-cheddar/CDN/master.m3u8
 http://stitcher.pluto.tv/stitch/hls/channel/5268abcd0ce20a8472000114/master.m3u8?sid=web24157571521&deviceType=web&deviceMake=web&deviceVersion=1&deviceId=web24163643069&appVersion=5.2.7&deviceDNT=web&deviceModel=web
```

Comment:
stitcher.pluto.tv

Reply:
That one is another version of Pluto TV's servers, that also works. The links we have in our playlists alternate between stitcher.pluto.tv and service-stitcher.pluto.tv

Reply:
We have only 6 links referring stitcher.pluto.tv and everuthing else is service-stitcher.pluto.tv already. And since stitcher ones try to serve US ads it's sometimes difficult to get content from the 1st attempt. That's why it was market as broken tonight

Reply:
There's the "&serverSideAds=false" to add to Stitcher links.

Reply:
When you do this manipulation, the server understands that it doesn't need to show advertisments on stream :)

![vlcsnap-2021-07-03-17h28m23s240](https://user-images.githubusercontent.com/30985701/124359180-25709280-dc24-11eb-8641-67173ad0e628.png)